CMakeLists.txt

```diff
@@ -1248,6 +1248,14 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
   include(${CMAKE_CURRENT_LIST_DIR}/cmake/mcuboot.cmake)
 endif()
 
+if(CONFIG_BUILD_WITH_TFM)
+  trusted_firmware_get_post_build_commands(TFM_POST_BUILD_COMMANDS)
```

Comment:
This is not the right location for this code.

The code here:
https://github.com/zephyrproject-rtos/zephyr/blob/a05bee5930b0d0786cc84243f3b4ef800c89b88d/boards/arm/mps2_an521/CMakeLists.txt#L24

seems to be depending on https://github.com/zephyrproject-rtos/trusted-firmware-m/blob/7316751a318fafdd1ae0d3594162dc0a1469157a/CMakeLists.txt#L156 that creates `tfm_sign.bin`
So having code in `boards/arm/mps2_an521/CMakeLists.txt` that is depending on this line to be executed first will break at some point.